src/ProjectTemplates/Web.ProjectTemplates/content/Worker-FSharp/.template.config/template.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    "Worker",
+    "Web"
+  ],
+  "name": "Worker Service",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty project template for creating a worker service.",
+  "groupIdentity": "Microsoft.Worker.Empty",
+  "precedence": "7000",
+  "identity": "Microsoft.Worker.Empty.FSharp.5.0",
+  "shortName": "worker",
+  "tags": {
+    "language": "F#",
+    "type": "project"
+  },
+  "sourceName": "Company.Application1",
+  "preferNameDirectory": true,
+  "guids": [
+    "53bc9b9d-9d6a-45d4-8429-2a2761773502"
```

Comment:
I just copied this GUID from the C# template, but if we need a new one I'll get a hot one from hotguids: http://www.secretgeek.net/hotGuids/index.htm